### PR TITLE
Update LurkingWifeMode.md

### PR DESCRIPTION
**Pull request by RiccardoMasutti**
Better formatting

Closes https://github.com/zkSNACKs/WasabiDoc/pull/185

### CODE REVIEW

**Comment by nopara73** [comment]
Untested ACK

File: docs/using-wasabi/LurkingWifeMode.md [hunk 1]
```diff
@@ -1,13 +1,22 @@
 ---
 {
   "title": "Lurking Wife Mode",
-  "description": "Hide sensitive information from physical observers and post screenshots."
+  "description": "A function that hides sensitive and critical information from physical observers and allows you to post screenshots without having to worry about your privacy. This is the Wasabi documentation, an archive of knowledge about the open-source, non-custodial and privacy-focused Bitcoin wallet for desktop."
```
Comment (yahiheb) [inline docs/using-wasabi/LurkingWifeMode.md]:
```suggestion
  "description": "A feature that hides sensitive and critical information from physical observers and allows you to post screenshots without worrying about your privacy. This is the Wasabi documentation, an archive of knowledge about the open-source, non-custodial and privacy-focused Bitcoin wallet for desktop."
```

File: docs/using-wasabi/LurkingWifeMode.md [hunk 2]
```diff
@@ -1,13 +1,22 @@
...
 }
 ---
 
+[[toc]]
+
+# Lurking Wife Mode (LWM)
+
+Lurking Wife Mode is a function of Wasabi Wallet that hides sensitive and critical information from physical observers and allows you to post screenshots without having to worry about your privacy.
```
Comment (yahiheb) [inline docs/using-wasabi/LurkingWifeMode.md]:
```suggestion
Lurking Wife Mode is an exclusive Wasabi Wallet feature that hides sensitive and critical information from physical observers and allows you to post screenshots without worrying about your privacy.
```

File: docs/using-wasabi/LurkingWifeMode.md [hunk 3]
```diff
@@ -1,13 +1,22 @@
...
+
 ![](https://i.imgur.com/4OgtaHP.png)
 
-- You can activate LWM from Settings or by clicking on your wallet balance. 
+## How to activate Lurking Wife Mode
```
Comment (yahiheb) [inline docs/using-wasabi/LurkingWifeMode.md]: Maybe this should be add to the FAQ?

Comment (RiccardoMasutti) [reply docs/using-wasabi/LurkingWifeMode.md]: You are right! Adding it in a couple of hours

File: docs/using-wasabi/LurkingWifeMode.md [hunk 4]
```diff
@@ -1,13 +1,22 @@
 ---
 {
   "title": "Lurking Wife Mode",
-  "description": "Hide sensitive information from physical observers and post screenshots."
+  "description": "A feature that hides sensitive and critical information from physical observers and allows you to post screenshots without worrying about your privacy. This is the Wasabi documentation, an archive of knowledge about the open-source, non-custodial and privacy-focused Bitcoin wallet for desktop."
```
Comment (lontivero) [inline docs/using-wasabi/LurkingWifeMode.md]: What about:
```suggestion
  "description": "A feature that hides sensitive and critical information from physical observers designed to allow you to post screenshots without worrying about leaking private details of your wallet. This is the Wasabi documentation, an archive of knowledge about the open-source, non-custodial and privacy-focused Bitcoin wallet for desktop."
```

**Comment by RiccardoMasutti** [comment]
#205

**Pull request merged**